XAMLator.Client.Net461/XAMLator.Client.Net461.csproj

```diff
@@ -0,0 +1,193 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What's the reason to convert XAMLator.Client into a library instead of a shared project? Is this a requirement for the Visual Studio addin?